tests/FakeItEasy.Analyzer.Tests/Helpers/CodeFixVerifier.Helper.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Simplification;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace TestHelper

```

Comment:
Can we change the namespace to `FakeItEasy.Analyzer.Tests.Helpers`? Same applies to all the other templated code.